invoicing/tests/test_process_payment_data.py

```diff
@@ -12,8 +12,8 @@
 from invoicing.tests.factories import ApartmentInstallmentFactory, PaymentFactory
 
 VALID_TEST_PAYMENT_DATA = """022121917199          12800     0000000000000000000000000000000000000000000000000000000000
-300000010700152221218221218730000077                           SAP ATestaaj1 00006658100  
-300000010700152221218221218730000077                           SAP BTestaaj1 00006658101  
```

Comment:
I think it would be better to leave these here too. Just extend the test to also check  the two new rows (payments 3 and 4) and fix the dates on line 46 and 48 to be `date(2022, 12, 18)`, since that's how there rows would be interpreted with YYMMDD date format.

Reply:
Or actually, since the two new rows are duplicating the same line twice, it would be better to just add one copy of it. (Doesn't add any value to have it twice).